.github/workflows/modmesh_install.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
         matrix:
-          os: [ubuntu-18.04, macos-latest]
+          os: [ubuntu-20.04, macos-latest]
```

Comment:
I would like to keep 18.04.  It's OK to add 20.04, but please do not drop the older arch.